.github/ISSUE_TEMPLATE/collection_proposal.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Propose a new collection
+about: Use this ticket when you are submitting new collection for review.
+title: 'collection: <Add your title>'
+labels: collection proposal
+assignees: ''
+
+---
+
+**Reviewer**: [Leave blank. web.dev team will assign a reviewer.]
+
+**Process**
+
+When you're ready, drop a link to your proposal doc in the 1st step below and
+check the box. A member of the web.dev team will assign this ticket to
+themselves and work as your reviewer to help you complete the remaining steps.
```

Comment:
```suggestion
a reviewer, who will help you complete the remaining steps.
```
To fix the number agreement error. Needs fixed in content proposal template also.